modules/i18n/src/translations/en-US/portals/email-providers.ts

```diff
@@ -0,0 +1,153 @@
+                            }
+                        }
+                    }
+                },
+                client_credential: {
+                    name: "Client Credential",
+                    properties: {
+                        tokenEndpoint: {
+                            label: "Token Endpoint",
+                            placeholder: "Token Endpoint",
+                            validations: {
+                                empty: "Token Endpoint is a required field."
+                            }
+                        },
+                        clientID: {
+                            label: "Client ID",
+                            placeholder: "Client ID",
+                            validations: {
+                                empty: "Client ID is a required field."
+                            }
+                        },
+                        clientSecret: {
+                            label: "Client Secret",
+                            placeholder: "Client Secret",
+                            validations: {
+                                empty: "Client Secret is a required field."
+                            }
+                        },
+                        scopes: {
+                            label: "Scopes",
+                            placeholder: "Scopes",
+                            validations: {
+                                empty: "Client Secret is a required field."
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        authenticationType: {
+            hint: {
+                create: "Once added, these secrets will not be displayed. You will only be able to reset them.",
+                update: "Once updated, these secrets will not be displayed. You will only be able to reset them again."
+            },
+            label: "Authentication",
+            placeholder: "Select Authentication Scheme",
+            validations: {
+                empty: "Authentication Scheme is a required field."
+            }
+        },
+        authenticationTypeDropdown: {
+            authProperties: {
+                clientID: {
+                    label: "Client ID",
+                    placeholder: "Client ID",
+                    validations: {
+                        required: "Client ID is a required field."
+                    }
+                },
+                clientSecret: {
+                    label: "Client Secret",
+                    placeholder: "Client Secret",
+                    validations: {
+                        required: "Client Secret is a required field."
+                    }
+                },
+                tokenEndpoint: {
+                    label: "Token Endpoint",
+                    placeholder: "Token Endpoint",
+                    validations: {
+                        required: "Token Endpoint is a required field."
```

Comment:
Let's keep the sentences in sentence case. Please check and fix the other places as well.

```suggestion
                        required: "Token endpoint is a required field."
```